src/styles.css

```diff
@@ -0,0 +1,28 @@
+#dice-box,
+#dice-box canvas {
+  position: fixed;
+  pointer-events: none;
+  z-index: 100;
```

Comment:
One minor issue on mobile: the dice sometimes land underneath the floating top bar. I think I'd rather have them appear overt top, so some z-index changes might need to be made.
![image](https://github.com/user-attachments/assets/acd1fc0e-a72d-4de3-aa52-c600dfad7b86)